lib/cdo/shared_constants/pd/shared_application_constants.rb

```diff
@@ -22,13 +22,6 @@ module SharedApplicationConstants
 
     COHORT_CALCULATOR_STATUSES = %w(
       accepted
-      accepted_not_notified
-      accepted_notified_by_partner
-      accepted_no_cost_registration
-      registration_sent
-      paid
     )
-
-    COHORT_VIEW_STATUSES = COHORT_CALCULATOR_STATUSES + ['withdrawn']
```

Comment:
This is part of Requirement 8 in https://docs.google.com/document/d/1HedK5_qh8-Yl6bFwfyU0ZB-UHgQK7l8-4trJhoGYI-8/edit#